.github/workflows/pkgdown.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - {os: macOS-latest,   r: 'release'}
+          - {os: ubuntu-20.04,   r: 'release', rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
```

Comment:
Part of me wants to keep mac for the aqua capabilities but it's not that important for bslib (it is for thematic, but I suppose we could always fully embrace ragg)

Reply:
(To put it another way, I'm fine with keeping ubuntu here)

Reply:
sounds good